statistics/histogram.go

```diff
@@ -139,8 +139,11 @@ func (hg *Histogram) MemoryUsage() (sum int64) {
 	if hg == nil {
 		return
 	}
+	if len(hg.Buckets) == 0 && len(hg.scalars) == 0 && hg.Bounds.Capacity() == 0 {
+		return
```

Comment:
`hg` still takes `EmptyHistogramSize` if it is not nil?

Reply:
Right, but we will think it as 0, the difference is acceptable.